core-blocks/test/full-content.js

```diff
@@ -6,14 +6,16 @@ import path from 'path';
 import { uniq, isObject, omit, startsWith, get } from 'lodash';
 import { format } from 'util';
 
+/**
+ * WordPress dependencies
+ */
+import { getBlockTypes, parse, serialize } from '@wordpress/blocks';
+
 /**
  * Internal dependencies
  */
-import { registerCoreBlocks } from '../library';
-import parse from '../api/parser';
-import { parse as grammarParse } from '../api/post.pegjs';
-import serialize from '../api/serializer';
-import { getBlockTypes } from '../api/registration';
+import { registerCoreBlocks } from '../';
+import { parse as grammarParse } from '../../blocks/api/post.pegjs';
```

Comment:
Just noting that it is imported from `blocks`.

Reply:
Another question is if those tests need to use low-level functions? Maybe there is a simpler way to achieve the same level of confidence.